src/database/sql/audio.py

```diff
@@ -412,7 +444,8 @@ def get_sad(self, audio_id: str) -> str:
         query = """
                 SELECT sad, sad_confidence, last_updated
                 FROM Audio
-                WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+                WHERE audio_release=:rel AND audio_side=:side
+                AND audio_track=:track
```

Comment:
If you want this AND `audio_track=:track` to be on a new line, please indent it.